HunterPie/Usecases/VerifyHunterPiePathUseCase.cs

```diff
@@ -1,11 +1,50 @@
 ﻿using HunterPie.Core.Client;
 using HunterPie.Core.Extensions;
+using System.IO;
+using System.Linq;
+using System;
 
 namespace HunterPie.Usecases;
 internal class VerifyHunterPiePathUseCase
 {
 
-    public static bool Invoke() => !ClientInfo.ClientPath.ToLowerInvariant()
-        .ContainsAny(new[] { "temp", "appdata" });
+    public static bool Invoke()
+    {
+        // Get the current execution directory
+        string executablePath = AppDomain.CurrentDomain.BaseDirectory;
```

Comment:
Should use `ClientInfo.ClientPath` here, they point to the same thing but it's better to not have multiple sources of the truth